hail/python/test/hailtop/utils/test_utils.py

```diff
@@ -181,3 +181,13 @@ def test_grouped_size_3_groups_7_elements():
     actual = list(grouped(3,['abc', 'def', 'ghi', 'jkl', 'mno', 'pqr', 'stu']))
     expected = [['abc', 'def', 'ghi'], ['jkl', 'mno', 'pqr'], ['stu']]
     assert actual == expected
+
+
+def test_unzip():
+    assert unzip([]) == ([], [])
+    assert unzip([(0, 'a')]) == ([0], ['a'])
+    assert unzip([(123,'')]) == ([123], [''])
+    assert unzip([(123, 'abc')]) == ([123], ['abc'])
+    assert unzip([(123, 456), ('abc', 'def')]) == ([123, 'abc'], [456, 'def'])
+    assert unzip([(123, 'abc'), (456, 'def'), (789, 'ghi')]) == ([123, 456, 789], ['abc', 'def', 'ghi'])
+    
```

Comment:
I think you might have four blank spaces on this line. Can you remove those? Thanks!

Reply:
To be clear, one empty line is great, but it shouldn't have any characters on it.

Reply:
Okay. I see, it had a space in the line